### PR TITLE
Update to KubeVirt 0.23.0

### PR DESCRIPTION
**Pull request by slintes**
Heads up, running `./hack/build-manifests.sh` removed the "KVM_EMULATION" env var from the hco operator in deploy/operator.yaml. Not sure if that is intentional?

### CODE REVIEW

File: deploy/operator.yaml [hunk 1]
```diff
@@ -19,8 +19,6 @@ spec:
       - command:
         - hyperconverged-cluster-operator
         env:
-        - name: KVM_EMULATION
```
Comment (rthallisey) [inline deploy/operator.yaml]: @arachmani it looks like this wasn't added to the CSV template.  Can you open a PR to add it?

**Comment by rthallisey** [comment]
@slintes without it, the prow job will fail.  Thanks for pointing it out.

File: deploy/operator.yaml (hunk 1 above)
Comment (rthallisey) [reply deploy/operator.yaml]: https://github.com/kubevirt/hyperconverged-cluster-operator/pull/343

**Comment by rthallisey** [comment]
#343 has been merged; can you rebase @slintes ?

**Comment by slintes** [comment]
> #343 has been merged; can you rebase @slintes ?

done, thx for the heads up

**Pull request merged**